src/components/header/header.js

```diff
@@ -16,7 +17,9 @@ export default function(){
 				<li><Link activeClassName="active" to="/support">Support</Link></li>
 			</ul></nav>
 			{/*<span>Search bar here</span>*/}
-			<CarbonAd />
+			
 		</header>
+		<CarbonAd />
```

Comment:
Moved the CarbonAd component out of header coz the child-node-elements of header are flex distributed with even spacing. Having the CarbonAd component inside the header element moved the previous siblings (the nav element) to the left coz now there were three child-elements inside the header.

And since that the CarbonAd element itse;f is fixed, it's positioning won't be affected at all 🤗